torch/csrc/xpu/Module.cpp

```diff
@@ -11,24 +11,30 @@
 #include <torch/csrc/utils/python_numbers.h>
 #include <torch/csrc/utils/python_strings.h>
 
+#ifndef WIN32
 #include <pthread.h>
+#endif
 
 using namespace torch;
 
 static bool in_bad_fork = false; // True for children forked after xpu init
```

Comment:
How do we address the bad fork on Windows?

Reply:
Align to other backends, it is not supported on Windows.

Reply:
Yes, aligned with other backends.